docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,161 @@ labels:
-- Search: Highlights spans containing the searched text.
-- Metadata: Various metadata about the trace.
+- **Header title** - Shows the name of the root span and trace ID.
+- **Search** - Highlights spans containing the searched text.
+- **Metadata** - Various metadata about the trace.
+
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-header.png" class="docs-image--no-shadow" max-width= "750px" caption="Trace view header" >}}
 
 ### Minimap
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-minimap.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view minimap" >}}
+**Minimap** displays a condensed view of the trace timeline. Drag your mouse over the minimap to zoom into a smaller time range. This also updates the main timeline, making it easier to view shorter spans
+When zoomed in, hovering over the minimap displays the **Reset selection** button, which resets the zoom.
 
-Shows condensed view or the trace timeline. Drag your mouse over the minimap to zoom into smaller time range. Zooming will also update the main timeline, so it is easy to see shorter spans. Hovering over the minimap, when zoomed, will show Reset Selection button which resets the zoom.
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-minimap.png" class="docs-image--no-shadow" max-width= "900px" caption="Trace view minimap example" >}}
 
-### Span filters
+### Timeline
 
-![Screenshot of span filtering](/media/docs/tempo/screenshot-grafana-tempo-span-filters-v10-1.png)
+Timeline shows list of spans within the trace. Each span row consists of the following components:
 
-Using span filters, you can filter your spans in the trace timeline viewer. The more filters you add, the more specific are the filtered spans.
+- **Expand children** - Expands or collapses all the children spans of the selected span.
+- **Service name** - Name of the service logged the span.
+- **Operation name** - Name of the operation that this span represents.
+- **Span duration bar** - Visual representation of the operation duration within the trace.
 
-You can add one or more of the following filters:
+Click anywhere on the span row to reveal span details.
 
-- Resource service name
-- Span name
-- Duration
-- Tags (which include tags, process tags, and log fields)
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-timeline.png" class="docs-image--no-shadow" max-width= "900px"  caption="Trace view timeline" >}}
 
-To only show the spans you have matched, you can press the `Show matches only` toggle.
+### Span details
 
-{{< youtube id="VP2XV3IIc80" >}}
+Traces are composed of one or more spans.
+A span is a unit of work within a trace that has a start time relative to the beginning of the trace, a duration and an operation name for the unit of work.
+It usually has a reference to a parent span (unless it’s the first span, the root span, in a trace).
```

Comment:
Using a comma instead of a parenthesis, since the toolkit suggestions minimizing parentheticals.
```suggestion
It usually has a reference to a parent span, unless it’s the first span, the root span, in a trace.
```